apps/site/lib/site_web/controllers/cms_controller.ex

```diff
@@ -43,6 +43,18 @@ defmodule SiteWeb.CMSController do
     |> handle_page_response(conn)
   end
 
+  def reset_cache_key(conn, %{"object" => object, "id" => id}) do
+    CMS.Repo.delete("/cms/#{object}/#{id}")
```

Comment:
question: is there a CMS PR for this, or is this built-in Drupal functionality that we're using?

Reply:
https://github.com/mbta/cms/pull/409